### PR TITLE
Mondido icons

### PR DESCRIPTION
**Pull request by robbanp**
Added icons for:
- Trustly
- Collector
- Mondido
- Invoice

### CODE REVIEW

**Comment by mondido-slava** [comment]
I've sent new PR with fixes: sizes, white background, readability.
https://github.com/activemerchant/payment_icons/pull/48

this PR can be closed.

**Comment by mondido-slava** [comment]
committed fixes: sizes,  background, readability. 
Please review. @AnthonyClark

File: db/payment_icons.yml [hunk 1]
```diff
@@ -242,3 +242,19 @@
   name: dailyyamazaki
   label: Daily Yamazaki
   group: convenience_stores
+-
+  name: mondido
+  label: Mondido
+  group: other
+-
+  name: collector_bank
+  label: Collector
```
Comment (AnthonyClark) [inline db/payment_icons.yml]: I don't have much context on this, but should the label by any chance be "Collector Bank"?

**Comment by robbanp** [comment]
Committed update: Collector Bank
Thanks for notice, Please review. @AnthonyClark

**Comment by bdewater** [comment]
Hey @robbanp and @mondido-slava I removed the invoice icon in 489f746b4f569e73a08d6331ebd3ab4295b035d2 because it was causing ImageMagick errors in Shopify, the font is not available for rendering on all developer machines, let alone servers. Could you please open a new PR with the font converted to paths so there is no dependancy? Thanks! 😄

**Pull request merged**